complex/cargo/list/src/lib.rs

```diff
@@ -374,6 +411,76 @@ pub unsafe extern "C" fn list_manager_create_item(manager: *mut ListManager, ite
     manager.create_item(&item);
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn list_manager_create_item_direct(manager: *mut ListManager, name: *const c_char, due_date: *const time_t) {
```

Comment:
This one can be renamed to `list_manager_create_item` and replace the old one I think.